modules/update-service-install-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+[id="update-service-install-web-console_{context}"]
+= Installing the {product-title} Update Service by using the web console
+
+You can use the {product-title} web console to install the {product-title} Update Service Operator.
+
+.Procedure
+
+To install the {product-title} Update Service Operator using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose *{product-title} Update Service* from the list of available Operators, and click *Install*.
+
+. FIXME: Does this apply? Ensure that the *All namespaces on the cluster* is selected under *Installation Mode*.
+
+. Select a namespace for *Installed Namespace*, such as the default FIXME-default-namespace.
+
+. FIXME: Does this apply? Select *Enable operator recommended cluster monitoring on this namespace*.
++
+This option sets the `openshift.io/cluster-monitoring: "true"` label in the Namespace object.
+
+. Select *{product-version}* as the *Update Channel*.
+
+. Select an *Approval Strategy*:
++
+** The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+** The *Manual* strategy requires a user with FIXME credentials to approve the Operator update.
```

Comment:
What credentials does the manual strategy require?

Reply:
I'm still not sure :(

Reply:
This is explained in https://docs.openshift.com/container-platform/4.6/operators/user/olm-installing-operators-in-namespace.html#olm-installing-operators-from-operatorhub_olm-installing-operators-in-namespace which we'll have already linked to as a prereq:
`If you select manual updates, when a newer version of an Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.`